templates/header.hbs

```diff
@@ -14,7 +14,7 @@
     <nav class="user-nav" id="user-nav" aria-label="{{t 'user_navigation'}}">
       <ul class="user-nav-list">
         <li>{{link 'community'}}</li>
-        <li><a href="/hc/en-us/services">Services</a></li>
+        <li>{{link 'service_catalog' class='service-catalog'}}</li>
```

Comment:
Nit: if we're not using the `service-catalog` class it'd be better not to include it